site/docs/v1/tech/guides/multi-factor-authentication.adoc

```diff
@@ -536,8 +537,42 @@ If you have other MFA methods that you'd like to use with FusionAuth, you have a
 
 include::docs/v1/tech/guides/_mfa-migration.adoc[]
 
+== Forcing MFA
+
+There are times when you might want to force a user to MFA before they ever get access to your applciation, or force every user of a certain type to MFA.
+
+These are two different situations.
+
+=== Forcing a User to Before Accessing an Application
```

Comment:
My opinion, the context feels a bit lost here, I know this is forcing a user to (MFA).. but it doesn't read very well I don't think.